src/System.Private.ServiceModel/tools/test/SelfHostWcfService/IWcfService.cs

```diff
@@ -13,7 +13,7 @@ namespace WcfService
     internal interface IWcfService
     {
         [OperationContract]
-        String EchoWithTimeout(String message);
+        String EchoWithTimeout(String message, TimeSpan timeout);
```

Comment:
the `timeout` parameter should be more verbose about which timeout we're talking about - e.g., `serviceOperationTimeout`.